### PR TITLE
Cleanup of the class

### PR DESCRIPTION
**Pull request by ameech**
- Added a namespace
- Updated comments to a better format
- Updated some private variables to protected so it can be extended

### CODE REVIEW

**Comment by dolfelt** [comment]
~~Let's ditch the `[]` for now since those are only available in PHP >=5.4.~~ Let's increase requirements to >=5.4

Also, could you add the PSR-4 configuration to the composer file?

**Comment by shadowhand** [comment]
@dolfelt PHP 5.3 is [EOL since August 2014](http://php.net/eol.php), do we really need to continue to support it?

File: src/Wheniwork.php [hunk 1]
```diff
@@ -1,4 +1,4 @@
-<?php
+<?php namespace Wheniwork;
```
Comment (shadowhand) [inline src/Wheniwork.php]: I think we need a sub-namespace for every project/library:

``` php
namespace Wheniwork\Api;
```

File: src/Wheniwork.php [hunk 2]
```diff
@@ -14,19 +14,23 @@
...
+    const METHOD_PUT    = 'put';
+    const METHOD_PATCH  = 'patch';
     const METHOD_DELETE = 'delete';
 
     private $api_key;
     private $api_token;
-    private $api_endpoint = 'https://api.wheniwork.com/2';
-    private $verify_ssl   = false;
+    protected $verify_ssl   = false;
+    protected $api_endpoint = 'https://api.wheniwork.com/2';
```
Comment (shadowhand) [inline src/Wheniwork.php]: Shouldn't this be a constant too?

**Comment by ameech** [comment]
Made a new PR. #3

**Pull request closed without merging**